app/views/items/show.html.erb

```diff
@@ -0,0 +1,22 @@
+<% flash.each do |key, message| %>
+  <div class="flash <%= key %>">
+    <%= message %>
+  </div>
+<% end %>
+
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Description</th>
+      <th>Unit Price</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><%= @item.name %></td>
+      <td><%= @item.description %></td>
+      <td><%= @item.unit_price %></td>
+    </tr>
+  </tbody>
+</table>
```

Comment:
This is really cool syntax and I have not seen the HTML table used before. Its cool that you are able to use this syntax to display a headers with the: 
- `item.name`
- `item.description`
- `item.unit_price`